Engines/Wine/Verbs/adobeair/script.js

```diff
@@ -28,16 +28,20 @@ Wine.prototype.adobeair = function () {
 /**
  * Verb to install adobeair
  */
-var verbImplementation = {
-    install: function (container) {
-        var wine = new Wine();
+class AdobeAirVerb {
```

Comment:
@plata @qparis this approach should work for all types of scripts that return a class which needs to be instantiated on the Java side.

Reply:
Can you try also with `Engine`. I tried myself but it didn't work. Either I did something wrong or I have a different issue here.

Reply:
Engine should work the same but will need more effort:

Ideally I would like to merge both the engine object and the engine implementation into a single JavaScript class. Afterwards we should be able to work with it using the same approach.

I don't want to do these changes in this PR, because I can't say yet what else in the scripts needs to be changed to make them work with a unified object.

Reply:
How did you try to make it work?

Reply:
Basically I tried to replace the `Java.extend` syntax by `class`. It should not be part of this pr (also the verbs). But in my opinion, we should really be sure that it will work. So what I would like is: one pr for each type (installer scripts, verbs, plugins, engines ...) with one adjusted script. Review that and then apply the changes to all scripts of that type. Then merge.

Reply:
I agree for the multi PR approach. On the Java side I think we will need a shared change (i.e. the semantic change of the `include` command). I would like to provide this as a separate PR which we can merge as is, because it shouldn't affect the status quo.

About your experiments:
What error did you get?

Reply:
Stuff like: `PolyglotMap cannot be used as array` (in Wine#run when the `ProcessBuilder` is called). No clue if it's related at all but before my changes this was working.